sample/build.gradle

```diff
@@ -57,4 +80,11 @@ dependencies {
 
     // Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+
+    androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.4.0'
```

Comment:
These can be extracted

Reply:
Addressed in 694d064736478f073fa510eb663c87a79120f863